docs/docs/cmd/planner/plan/plan-set.md

```diff
@@ -17,10 +17,13 @@ m365 planner plan set [options]
 : Title of the plan. Specify either `id` or `title` but not both.
 
 `--ownerGroupId [ownerGroupId]`
-: ID of the group to which the plan belongs. Specify either `ownerGroupId` or `ownerGroupName` when using `title`.
+: ID of the group to which the plan belongs. Specify either `ownerGroupId`, `ownerGroupName` or `rosterId` when using `title`.
 
 `--ownerGroupName [ownerGroupName]`
-: Name of the Group to which the plan belongs. Specify either `ownerGroupId` or `ownerGroupName` when using `title`.
+: Name of the Group to which the plan belongs. Specify either `ownerGroupId`, `ownerGroupName` or `rosterId` when using `title`.
+
+`--rosterId [rosterId]`
```

Comment:
this is a bit tricky 🤔
@milanholemans maybe you could recheck this one with me 👍.
As I understand we are suppose to give the possibility to get roster plan by its `id` but this command already has the option `id` to get plan by it. I understand that currently it is different endpoint when we go for group plans or roster but I have a feeling the user should specify either `id` or `rosterId` 🤔. 
Currently how it is described we should specify `rosterId` when also giving the plan `title` which is like two options to specify the same thing right? 
What is also tricky is that I think we currently may not get roster plans by `title` so if user would like to set changes for roster he or she may only use `rosterId`. Is that also correct or am I missing something?

Reply:
If the user specifies a plan ID of a plan that is contained in a Roster, we can use the `id` option like we do now.
The thing is that at least right now you can only add 1 plan to a roster. This means that when you specify the `rosterId`, we have enough info to know which plan we are talking about.
The more strange part is, when you list all plans in a Roster, you're getting an array with 1 plan.
I have no idea whether they plan to add support for more plans within a Roster, but for now I'd say that a user should specify at least `id`, `title` or `rosterId`.

Reply:
agreed ✅ (I think that I what I tried to explain but my thinking was a bit to chaotic 😜). 
Now it seems we may specify both `title` and `rosterId` but the second could be enough right?
What if we would validate that the user may specify either `id`, `title` or `rosterId` and when `title` will be used the user needs to additionally specify `ownerGroupId` or `ownerGroupName` . @milanholemans would you agree on this kind of approach?
Currently we validate `rosterId` together with `ownerGroupId` or `ownerGroupName` 🤔.

Reply:
I think your suggested approach is the right way forward indeed!

Reply:
@nicodecleyre what do you think? Could we adapt the current approach to align with the above discussion?

Reply:
@Adam-it , looks good to me! Let's do it with that approach 💪! Thanks @milanholemans for your insights!